packages/design-system-web/src/components/geov-list-item-nested-properties/geov-list-item-nested-properties.tsx

```diff
@@ -201,9 +201,11 @@ export class GeovListItemNestedProperties {
               <geov-time-span sparqlEndpoint={this.sparqlEndpoint} entityUri={getTimeSpanUri(this.entityUri)}></geov-time-span>
             </div>
             <div class={'entityLabelContainer'}>
-              <a href={this.prepareUrl(this.entityUri)} target="_blank" class="entityLink">
-                {rdfsLabelProp?.object?.value}
-              </a>
+              <h2>
+                <a href={this.prepareUrl(this.entityUri)} target="_blank" class="entityLink">
+                  {rdfsLabelProp?.object?.value}
+                </a>
+              </h2>
             </div>
```

Comment:
We have to adjust this in the context of PR #109

Generally I would rather keep the font sizes as small as possible 
```suggestion
              <a href={this.prepareUrl(this.entityUri)} target="_blank" class="entityLink">
```